evap/staff/importers.py

```diff
@@ -815,12 +896,34 @@ def process_contributors(self, evaluation, test_run, user_list):
             for user in users_to_add:
                 order = Contribution.objects.filter(evaluation=evaluation).count()
                 Contribution.objects.create(evaluation=evaluation, contributor=user, order=order)
-            msg = format_html(_("{} contributors added to the evaluation {}:"), len(users_to_add), evaluation.full_name)
+
+        if len(users_to_add) == 0:
+            if test_run:
+                msg = format_html(_("No contributors would be added to the evaluation {}."), evaluation.full_name)
+            else:
+                msg = format_html(_("No contributors added to the evaluation {}."), evaluation.full_name)
         else:
-            msg = format_html(
-                _("{} contributors would be added to the evaluation {}:"), len(users_to_add), evaluation.full_name
-            )
-        msg += create_user_list_html_string_for_message(users_to_add)
+            if test_run:
+                msg = format_html(
+                    _("{sentence}: {list}"),
+                    sentence=ngettext(
+                        "{user_count} contributor would be added to the evaluation {name}",
+                        "{user_count} contributors would be added to the evaluation {name}",
+                        len(users_to_add),
+                    ).format(user_count=len(users_to_add), name=evaluation.full_name),
+                    list=create_user_list_html_string_for_message(users_to_add),
+                )
+            else:
+                msg = format_html(
+                    _("{sentence} added to the evaluation {name}: {list}"),
```

Comment:
Uniformly with the other places, this could be a `ngettext` on the "outer" translation, i.e. translate between:
* {count} contributor added to the evaluation {name}: {list}
* {count} contributors added to the evaluation {name}: list{}

(if we don't, `sentence` is a weird formatter string, we should use `contributor_object`)